features/nvstore/source/nvstore_int_flash_wrapper.cpp

```diff
@@ -20,7 +20,9 @@
 
 #include "nvstore_int_flash_wrapper.h"
 #include "nvstore_shared_lock.h"
+#ifdef DEVICE_FLASH
```

Comment:
it might be more readable if this file is protected entirely by #ifdef DEVICE_FLASH ? as this entire implementation depends on it?

btw, the include should be fine without the protection